### ISSUE TITLE
[Feature Request] Characters resuming/intiating dialogue with you

### ISSUE THREAD

**Issue by xNissX233** [SillyTavern/SillyTavern#1100]
*This is a copy paste from a request sent to character.ai long time ago. Since they ignored it, what about implementing it in SillyTavern?*
Imagine you're browsing the website and suddenly a pop-up at the bottom/top right/left of you screen showed up, and it was of a character you haven't spoken to in a while, saying "hey, how are things going? did you forget about me?". Obviously, it wouldn't have to be this cheesy, and ideally it'd be optional. And, something else along these lines, would be selecting characters that can actually initiate dialogue with you, although I'm guessing this one would be a bit tougher, since characters only have introduction lines, and not "hello"s. I'm assuming this wouldn't be too hard to rectify, though.

So, in summary, characters having the ability to auto send you a message if you don't message them in a while (with SillyTavern running time or with a timer comparing the last message with the actual date, I think this last one is already a variable that exists within SillyTavern)

**Comment by Cohee1207** [comment]
It's unlikely to be implemented by ST team. Someone could make a third party plugin if they want. All prompt building is done on the frontend, so they have to overcome technical limitations of background browser tabs.

**Comment by user177013** [comment]
How about sending auto messages in chat? when we chat with human, we don't always take the initiative and wait for our conversation partner to respond, it instead goes both ways and that's what makes it more engaging.

I think it'd be more like auto continue message, maybe with toggle to set the probabilities?

Maybe with some kind of auto topic generation and random temperature... would help when a chat goes in a rut lol.

**Comment by xNissX233** [comment]
> How about sending auto messages in chat? when we chat with human, we don't always take the initiative and wait for our conversation partner to respond, it instead goes both ways and that's what makes it more engaging.
> 
> I think it'd be more like auto continue message, maybe with toggle to set the probabilities?
> 
> Maybe with some kind of auto topic generation and random temperature... would help when a chat goes in a rut lol.

I don't see why would be difficult to implement something like that tbh. Just a timer if you enable the feature and if no message arrives in X minutes, trigger a message with context "hasn't talked to you in 1 hour" or something like that
Also the temperature idea to trigger random topics is cool.

**Comment by Cohee1207** [comment]
@city-unit is working on it, check out his PR for the preview version: #1132
